hidimstat/estimator/Dnn_learner_single.py

```diff
@@ -241,6 +241,7 @@ def fit(self, X, y=None):
         loss = np.array(res_ens[4])
 
         if self.n_ensemble == 1:
+            raise Warning("The model can't be fit with n_ensemble = 1")
```

Comment:
@jpaillard Can you check if it's correct that without multiple ensembles, there is not fitting?

Reply:
I think there is a fitting, as suggested by the function call `joblib_ensemble_dnnet` (btw should this be a private function ? Should we integrate it in the same module for clarity ?) 

The following lines select the `n_ensemble` best models (which is useless when `n_ensemble`==1). 

I would suggest to have a function `_keep_n_ensemble` that is called line 243 and gathers the code until line 261 to manage this distinction.

Reply:
See the Issue # 75. This will be modified in the future.